en/step_9.md

```diff
@@ -4,8 +4,8 @@ Now you've got started with the Camera Module, you could try adding GPIO control
 
 - [Push button stop-motion](https://projects.raspberrypi.org/en/projects/push-button-stop-motion/)
 - [Minecraft photobooth](https://projects.raspberrypi.org/en/projects/minecraft-photobooth/)
-- [Tweeting Babbage](https://projects.raspberrypi.org/en/projects/tweeting-babbage/)
-- [Parent detector](https://projects.raspberrypi.org/en/projects/parent-detector/)
+- [Tweeting Babbage](https://learning-admin.raspberrypi.org/en/projects/rpi-python-tweeting-babbage)
```

Comment:
This isn't a public facing url.

Reply:
I had to pull these changes into draft as they had been done directly in master.